openhtf/output/web_gui/__init__.py

```diff
@@ -315,7 +316,7 @@ def initialize(self, port):  # pylint: disable=arguments-differ
     def get(self):
       self.render('index.html', host=socket.gethostname(), port=self.port)
 
-  def __init__(self, discovery_interval_s, disable_discovery, http_port,
+  def __init__(self, discovery_interval_s, disable_discovery, http_port, plugins = [],
```

Comment:
plugins should default to None. See https://google.github.io/styleguide/pyguide.html?showone=Default_Argument_Values#Default_Argument_Values